Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md

```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
 
 ### Area configuration
 
+An area 
+
 ![Block Grid - Area Configuration](images/BlockGridEditor_AreasConfiguration.png)
 
-* **Alias** - The alias is printed by `GetBlockGridHTML()`. Use the alias to target the Element representing the area.
+* **Alias** - The alias is used to identify this Area.
+It begin printed by `GetBlockGridHTML()` And used as name for the Area slot in Custom Views.
+The alias is also available for CSS Selectors to target the HTML-Element representing a Area.
```

Comment:
This is a list, right? Should each new line here be a new list item, or is the text meant to be part of the "Alias" list item? 🤔

Reply:
Oh,. yeah, its a list. so this was additional tekst about the alias.